airbyte-integrations/connectors/source-hubspot/source_hubspot/spec.json

```diff
@@ -83,6 +83,12 @@
             }
           }
         ]
+      },
+      "unset_dynamic_fields": {
```

Comment:
probable skip_dynamic_fields  is better, lets discuss

Reply:
changed: unset_dynamic_fields -> skip_dynamic_fields